presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/BaseRuleTest.java

```diff
@@ -13,19 +13,31 @@
  */
 package com.facebook.presto.sql.planner.iterative.rule.test;
 
+import com.facebook.presto.tpch.TpchConnectorFactory;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 
 import static io.airlift.testing.Closeables.closeAllRuntimeException;
 
 public abstract class BaseRuleTest
 {
+    private final boolean tpchPredicatePushdownEnabled;
     private RuleTester tester;
 
+    protected BaseRuleTest()
+    {
+        this(TpchConnectorFactory.DEFAULT_PREDICATE_PUSHDOWN_ENABLED);
```

Comment:
from testing point of view, using explicit `false` (or whatever this is) would be better.
i.e., if tpch connector decides to change its default, should presto tests change their behavior and coverage?

Reply:
I personally do not like this flag at all. Lets leave it as it is for now. I will post a PR with removing this flag as a follow up PR.